arex-web-core/src/main/java/com/arextest/web/core/business/config/application/ApplicationOperationConfigurableHandler.java

```diff
@@ -33,7 +46,43 @@ public boolean insert(ApplicationOperationConfiguration configuration) {
         return super.insert(configuration);
     }
 
+    @Override
+    public boolean update(ApplicationOperationConfiguration configuration) {
```

Comment:
how to update the contract of dependency? is it need an singleton operation to overwrite the contract of operation and dependency